apps/changelog/constants/defaults.py

```diff
@@ -0,0 +1,5 @@
+HTK_CHANGELOG_FILE_PATH = 'CHANGELOG.md'
+HTK_CHANGELOG_SLACK_CHANNEL_RELEASES = '#release-notes'
+HTK_CHANGELOG_SLACK_FULL_LOG_MESSAGE = (
```

Comment:
`HTK_CHANGELOG_VIEW_IN_WEB_URL = None`

Message should be:
`The full change log can be found at: <url>`

Code would look something like:

```
web_url = htk_setting(HTK_CHANGELOG_VIEW_IN_WEB_URL)
if web_url:
    buf.append(The full change log can be found at: {}`.format(web_url))
```

Reply:
Done